pkg/cgroup/v1/set_test.go

```
@@ -0,0 +1,145 @@
+/*
+Copyright 2021 Andy Dalton
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cgroup_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/adalton/teleport-exercise/pkg/adaptation/os"
+	"github.com/adalton/teleport-exercise/pkg/adaptation/os/ostest"
+	"github.com/adalton/teleport-exercise/pkg/cgroup/v1"
+	"github.com/adalton/teleport-exercise/pkg/cgroup/v1/cgrouptest"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_Set_Create_Success(t *testing.T) {
+	jobId, _ := uuid.Parse("0b5183b8-b572-49c7-90c4-fffc775b7d7b")
```

Use uuid.MustParse instead of swallowing the error.